packages/segmented-control/src/SegmentedControl.spec.tsx

```diff
@@ -28,8 +28,6 @@ const renderNewContainer = () => {
   const banana = getByText(rendered.container, 'Banana')
     .parentElement as Element;
 
-  userEvent.tab();
```

Comment:
It this isn't needed, that's fantastic!

Reply:
Yeah, it took me a while this was the reason the tests were failing. It was making the first item focused but the first test is checking if the first item is not in focus.